src/ngraph/op/gather.hpp

```diff
@@ -58,6 +58,7 @@ namespace ngraph
             {
             public:
                 static constexpr NodeTypeInfo type_info{"Gather", 1};
+                static const int64_t AXIS_NOT_SET_VALUE = std::numeric_limits<int64_t>::max();
```

Comment:
Might need `const int64_t Gather::AXIS_NOT_SET_VALUE;` in the .cpp file for some compilers.